web/styles/left_sidebar.css

```diff
@@ -59,10 +59,36 @@ $before_unread_count_padding: 3px;
 }
 
 #stream_filters,
-#global_filters {
+#left_sidebar_navigation_list {
     margin-right: 12px;
 }
 
+.left-sidebar-navigation-menu-icon {
+    display: none;
+}
+
+#views_label_container {
+    cursor: pointer;
+    padding: 0 10px 0 6px;
```

Comment:
It. would be good to document the origin of all of these different pixel values, here and throughout.